dev/build.py

```diff
@@ -56,7 +56,7 @@ def main():
 
     if args.sha:
         # If build succeeds, there should be one wheel in the dist directory
-        wheel = next(Path("dist").glob("mlflow-*.whl"))
+        wheel = next(Path("dist").glob("mlflow*.whl"))
```

Comment:
The wheel path starts with `mlflow-` or `mlflow_skinny-`. `mlflow-*.whl` only matches `mlflow-`.